README.md

```diff
@@ -334,11 +334,18 @@ const app = new App({
 });
 
 const middleware = createNodeMiddleware(app);
-
-require("http").createServer(middleware).listen(3000);
+createServer(async (req, res) => {
```

Comment:
can you add the require statement for `createServer` on top if you remove it here?